licenses/TEMPLATE_LICENSE.rst

```diff
@@ -0,0 +1,31 @@
+This project is based upon the Astropy package template
```

Comment:
We should do a find and replace in the open astronomy package template for "astropy" lol